lua/astrocommunity/recipes/auto-session-restore/init.lua

```diff
@@ -0,0 +1,35 @@
+local read_from_stdin = false
```

Comment:
Hi, I mentioned it in the description.

"detect if nvim has been started with stdin redirected, like stream | nvim or like export MAPAGER='nvim +Man!'; man man."

I also mention I have taken the method to detect from vim-workspace plugin.

The issue is that if the snippet from documentation is used and `MANPAGER='nvim +Man!'` is set, it is _not possible_ to open a man page. Instead of opening a man page, the session is loaded.

Reply:
Then it's not sure if its something that should be in the recipes, those are for the examples from the docs. @mehalter has the final word on this

Reply:
this variable makes sense, I do think there is potentially a more performant method. But also this should totally be in the recipes!